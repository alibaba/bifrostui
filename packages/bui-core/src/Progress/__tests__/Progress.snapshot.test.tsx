import { snapshotTest } from 'testing';

describe('Progress demo snapshot', () => {
  snapshotTest('Progress');
});
