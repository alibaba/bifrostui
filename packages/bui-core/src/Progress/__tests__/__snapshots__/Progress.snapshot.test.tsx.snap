// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Progress demo snapshot Progress demo snapshot 0 1`] = `
<div
  className="bui-progress"
>
  <div
    className="bui-progress-inner"
    style={
      Object {
        "background": undefined,
      }
    }
  >
    <div
      className="bui-progress-bg"
      style={
        Object {
          "background": undefined,
          "height": undefined,
          "width": "40%",
        }
      }
    />
  </div>
</div>
`;

exports[`Progress demo snapshot Progress demo snapshot 0 2`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "5px",
    }
  }
>
  <div
    className="bui-progress"
  >
    <div
      className="bui-progress-inner"
      style={
        Object {
          "background": undefined,
        }
      }
    >
      <div
        className="bui-progress-bg"
        style={
          Object {
            "background": "var(--bui-color-primary)",
            "height": undefined,
            "width": "40%",
          }
        }
      />
    </div>
  </div>
  <div
    className="bui-progress"
  >
    <div
      className="bui-progress-inner"
      style={
        Object {
          "background": undefined,
        }
      }
    >
      <div
        className="bui-progress-bg"
        style={
          Object {
            "background": "var(--bui-color-info)",
            "height": undefined,
            "width": "50%",
          }
        }
      />
    </div>
  </div>
  <div
    className="bui-progress"
  >
    <div
      className="bui-progress-inner"
      style={
        Object {
          "background": undefined,
        }
      }
    >
      <div
        className="bui-progress-bg"
        style={
          Object {
            "background": "var(--bui-color-success)",
            "height": undefined,
            "width": "30%",
          }
        }
      />
    </div>
  </div>
  <div
    className="bui-progress"
  >
    <div
      className="bui-progress-inner"
      style={
        Object {
          "background": undefined,
        }
      }
    >
      <div
        className="bui-progress-bg"
        style={
          Object {
            "background": "var(--bui-color-warning)",
            "height": undefined,
            "width": "80%",
          }
        }
      />
    </div>
  </div>
  <div
    className="bui-progress"
  >
    <div
      className="bui-progress-inner"
      style={
        Object {
          "background": undefined,
        }
      }
    >
      <div
        className="bui-progress-bg"
        style={
          Object {
            "background": "var(--bui-color-vip)",
            "height": undefined,
            "width": "90%",
          }
        }
      />
    </div>
  </div>
</div>
`;

exports[`Progress demo snapshot Progress demo snapshot 0 3`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "flex-start",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "10px",
    }
  }
>
  <div
    className="bui-progress"
  >
    <div
      className="bui-progress-inner"
      style={
        Object {
          "background": undefined,
        }
      }
    >
      <div
        className="bui-progress-bg"
        style={
          Object {
            "background": "var(--bui-color-warning)",
            "height": 10,
            "width": "60%",
          }
        }
      />
    </div>
  </div>
  <div
    className="bui-progress"
  >
    <div
      className="bui-progress-inner"
      style={
        Object {
          "background": undefined,
        }
      }
    >
      <div
        className="bui-progress-bg"
        style={
          Object {
            "background": "var(--bui-color-warning)",
            "height": 15,
            "width": "60%",
          }
        }
      />
    </div>
  </div>
</div>
`;

exports[`Progress demo snapshot Progress demo snapshot 0 4`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "flex-start",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "10px",
    }
  }
>
  <div
    className="bui-progress"
  >
    <div
      className="bui-progress-inner"
      style={
        Object {
          "background": undefined,
        }
      }
    >
      <div
        className="bui-progress-bg"
        style={
          Object {
            "backgroundImage": "linear-gradient(to right, var(--bui-color-red), var(--bui-color-red-light))",
            "height": undefined,
            "width": "60%",
          }
        }
      />
    </div>
  </div>
  <div
    className="bui-progress"
  >
    <div
      className="bui-progress-inner"
      style={
        Object {
          "background": undefined,
        }
      }
    >
      <div
        className="bui-progress-bg"
        style={
          Object {
            "backgroundImage": "linear-gradient(to right, var(--bui-color-orange) 0%, var(--bui-color-orange-light) 100%)",
            "height": undefined,
            "width": "90%",
          }
        }
      />
    </div>
  </div>
</div>
`;

exports[`Progress demo snapshot Progress demo snapshot 0 5`] = `
<div
  className="bui-progress"
>
  <div
    className="bui-progress-inner"
    style={
      Object {
        "background": "var(--bui-color-red-light)",
      }
    }
  >
    <div
      className="bui-progress-bg"
      style={
        Object {
          "background": undefined,
          "height": undefined,
          "width": "60%",
        }
      }
    />
  </div>
</div>
`;
